File: solutions/python/001-fizzbuzz/solution_test.py
from solution import fizz_buzz, fizz_buzz_range

assert fizz_buzz(1) == "1"
assert fizz_buzz(3) == "Fizz"
assert fizz_buzz(5) == "Buzz"
assert fizz_buzz(15) == "FizzBuzz"
assert fizz_buzz(7) == "7"
assert fizz_buzz(30) == "FizzBuzz"
r = fizz_buzz_range(15)
assert r.startswith("1 2 Fizz 4 Buzz")
assert r.endswith("FizzBuzz")
print("ok")

File: solutions/javascript/001-fizzbuzz/solution.test.mjs
import test from "node:test";
import assert from "node:assert/strict";
import { fizzBuzz, fizzBuzzRange } from "./solution.mjs";

test("fizzBuzz cases", () => {
  assert.equal(fizzBuzz(1), "1");
  assert.equal(fizzBuzz(3), "Fizz");
  assert.equal(fizzBuzz(5), "Buzz");
  assert.equal(fizzBuzz(15), "FizzBuzz");
  assert.equal(fizzBuzz(7), "7");
  assert.equal(fizzBuzz(30), "FizzBuzz");
});

test("range 1..15", () => {
  const r = fizzBuzzRange(15);
  assert.ok(r.startsWith("1 2 Fizz 4 Buzz"));
  assert.ok(r.endsWith("FizzBuzz"));
});
